api_tests/nodes/views/test_node_links_detail.py

```diff
@@ -55,8 +55,8 @@ def test_returns_public_node_pointer_detail_logged_in(self):
 
     def test_returns_private_node_pointer_detail_logged_out(self):
         res = self.app.get(self.private_url, expect_errors=True)
-        assert_equal(res.status_code, 401)
-        assert_in('detail', res.json['errors'][0])
+        assert_equal(res.status_code, 200)
+        assert_in('errors', res.body)
```

Comment:
Are the errors from the embedding of the node detail information failing? If so, could you do a second test for each of these conditions where it verifies the correct error is in the correct place (embeds.target_node.errors == 'you don't have permission' or whatever)?